### PR TITLE
Feature/dp 7155 search fixes

### PR DESCRIPTION
**Pull request by chrisvanmook**

### CODE REVIEW

File: src/normalizations/cms/useNormalizedCMSResults.js [hunk 1]
```diff
@@ -19,12 +18,11 @@ const useNormalizedCMSResults = (aggregatedData, type) => {
...
-        const { href } = linkAttributesFromAction(
-          special_type
-            ? EDITORIAL_DETAIL_ACTIONS[type](uuid, special_type, slug)
-            : EDITORIAL_DETAIL_ACTIONS[type](uuid, slug || useSlug(title)),
-        )
+        const to = special_type
+          ? EDITORIAL_DETAIL_ACTIONS[type](uuid, special_type, slug)
+          : EDITORIAL_DETAIL_ACTIONS[type](uuid, slug || useSlug(title))
 
+        console.log(to)
```
Comment (royderks) [inline src/normalizations/cms/useNormalizedCMSResults.js]: Remove

File: src/app/components/EditorialCard/EditorialCard.jsx [hunk 2]
```diff
@@ -35,6 +36,12 @@ const StyledLink = styled(Link)`
   }
 `
 
+const StyledLinkWrapper = ({ children, ...otherProps }) => (
+  <Link $as={StyledRouterLink} gutterBottom={3} {...otherProps}>
```
Comment (aditudorache) [inline src/app/components/EditorialCard/EditorialCard.jsx]: why 3? Our spacing unit is 4 right?

File: src/app/components/EditorialResults/EditorialResults.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import { Button, CardContainer, Heading, svgFill, themeColor } from '@datapunt/asc-ui'
+import LoadingIndicator from '../../../shared/components/loading-indicator/LoadingIndicator'
+import { EDITORIAL_TITLES } from '../../pages/EditorialOverviewPage/constants'
+import EditorialCard from '../EditorialCard'
+
+const EditorialCardContainer = styled(CardContainer)`
+  padding: 0;
+`
+
+const StyledButton = styled(Button)`
```
Comment (royderks) [inline src/app/components/EditorialResults/EditorialResults.jsx]: We can still look into added a new variant for the Button to `asc-ui`

File: src/app/components/EditorialResults/EditorialResults.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  border-color: ${themeColor('tint', 'level7')};
+  color: ${themeColor('tint', 'level7')};
+  background: #fff;
```
Comment (aditudorache) [inline src/app/components/EditorialResults/EditorialResults.jsx]: tint.level1

File: src/app/components/EditorialCard/EditorialCard.jsx (hunk 2 above)
Comment (royderks) [reply src/app/components/EditorialCard/EditorialCard.jsx]: Yes, so 3 *4 = 12px

Comment (chrisvanmook) [reply src/app/components/EditorialCard/EditorialCard.jsx]: sharrup!

**Pull request merged**